src/Tables.jl

```diff
@@ -118,8 +118,7 @@ Obviously every table type is different, but via a combination of `Tables.rows`
 abstract type Table end
 
 # default definitions
-istable(x::T) where {T} = istable(T) || TableTraits.isiterabletable(x) === true ||
-    TableTraits.isiterabletable(x) === missing
+istable(x::T) where {T} = istable(T) || TableTraits.isiterabletable(x) === true || Base.isiterable(T)
```

Comment:
Is the ``Base.isiterable(T)`` part really a good idea? At that point ``istable`` returns ``true`` for pretty much anything, and for lots of things where it is entirely clear that they are *not* a table. I guess I don't really understand what purpose ``istable`` has at that point...